draft-ietf-quic-multipath.md

```diff
@@ -1028,16 +1024,16 @@ ACK_MP frame is formatted as shown in {{fig-ack-mp-format}}.
 Compared to the ACK frame specified in {{QUIC-TRANSPORT}}, the following
 field is added.
 
-Path Identifier:
-: The identifier of the path to identify the packet number
+Receiver's Destination Connection ID Sequence Number:
+: The sequence number of the Connection ID identifying the packet number
   space of the 1-RTT packets which are acknowledged by the ACK_MP frame.
 
```

Comment:
I think we have an issue of consistency. In the Abandon and Path Status frames, the DCID field is called "Sender's DCID". Here, it is called "Receiver's DCID". Yet, they refer to exactly the same value. My suggestion:

* Remove "Sender's" from the DCID descriptions in PATH_ABANDON and PATH_STATUS
* Remove "Receiver's" from the DCID descriptions in ACK_MP
* Otherwise, keep the text as is, because the text is correct.

Reply:
No they do not refer to the same value. The Sender's CID is the one used when you sent a packet. The Receivers's CID is the one used by the other end to send - hence when you receive packets.

Reply:
Actually the PATH_ABANDON and PATH_STATUS frame includes the sequence number of the DCID of the sender of the frame, while in ACK_MP the sequence number corresponds to the DCID used by its peer, hence the (verbose) name convention. But if this introduces confusion, I can remove "Sender's" and "Receiver's" mentions.

Reply:
Actually we did a mistake here when I remove the type field. The path abandon frame should also have the receivers cid as you tell the other end to not use this for sending data anymore. Same I probably true for the path status frame.

Reply:
I think aligning ACK, ABANDON and STATUS makes sense. ABANDON is in many respects just the symmetric of the ACK. For example, a node receives data on a cellular path that charges per byte when a "free" path is available. The node will send the ABANDON immediately, telling its peer to "stop doing that." Using the DCID "used by the peer to send 1-RTT packets" makes a lot of sense.

Reply:
Yeah, this would make sense. Let me update the text and the examples accordingly.